socorro/siglists/signature_sentinels.txt

```diff
@@ -5,5 +5,6 @@
 # Note: no regular expressions here.
 
 _purecall
+_sigtramp
```

Comment:
This will cause signature generation to start the signature at this frame if it exists. In cases where this shows up, everything above it is crash reporter frames.

This is a good example of that: https://crash-stats.mozilla.com/report/index/2e177d22-8236-40dc-88f7-2e5b10171018

Reply:
To confirm my understanding, signature generation will start here but will not include this frame in the final signature, since this is also in the irrelevant list?

Reply:
Yes.